Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -51,18 +51,27 @@ NativeReanimatedModule::NativeReanimatedModule(std::shared_ptr<CallInvoker> jsIn
                                                std::shared_ptr<Scheduler> scheduler,
                                                std::unique_ptr<jsi::Runtime> rt,
                                                std::function<void(std::function<void(double)>)> requestRender,
-                                               std::function<void(jsi::Runtime&, int, const jsi::Object&)> propUpdater):
+                                               std::function<void(jsi::Runtime&, int, const jsi::Object&)> propUpdater,
+                                               ScrollToFunction scrollToFunction,
```

Comment:
I think it is about the time we make some sort of interface that'd provide methods that have separate platform-dependent implementation. Was something I wanted to do from the startt. But as we are getting more methods here this starts to look a bit obscure